game_eggs/minecraft/proxy/java/viaaas/egg-v-i-aaa-s.json

```diff
@@ -4,47 +4,48 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2021-06-28T11:55:37-04:00",
+    "exported_at": "2021-10-16T11:26:14-03:00",
     "name": "VIAaaS",
     "author": "regulad@outlook.com",
     "description": "VIAaaS - ViaVersion as a Service - Standalone ViaVersion proxy",
     "features": null,
     "images": [
-        "ghcr.io\/pterodactyl\/yolks:java_16"
+        "ghcr.io\/pterodactyl\/yolks:java_16",
+        "ghcr.io\/pterodactyl\/yolks:java_17"
     ],
     "file_denylist": [],
     "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}} -sslPort={{WEBSERVER_PORT}}",
     "config": {
         "files": "{\r\n    \"config\/viaaas.yml\": {\r\n        \"parser\": \"yaml\",\r\n        \"find\": {\r\n            \"port\": \"{{server.build.default.port}}\",\r\n            \"bind-address\": \"0.0.0.0\"\r\n        }\r\n    }\r\n}",
-        "startup": "{\r\n    \"done\": \"Application started:\"\r\n}",
+        "startup": "{\r\n    \"done\": \"Application started in \"\r\n}",
         "logs": "{\r\n    \"custom\": false,\r\n    \"location\": \"logs\/latest.log\"\r\n}",
         "stop": "end"
     },
     "scripts": {
         "installation": {
-            "script": "#!\/bin\/bash\r\n# VIAaaS Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\napt update\r\napt install -y curl\r\n\r\ncd \/mnt\/server\r\n\r\ncurl -Lf -o ${SERVER_JARFILE} \"https:\/\/jitpack.io\/com\/github\/ViaVersion\/VIAaaS\/master-SNAPSHOT\/VIAaaS-master-SNAPSHOT-all.jar\"",
-            "container": "debian:buster-slim",
-            "entrypoint": "bash"
+            "script": "#!\/bin\/ash\r\n# VIAaaS Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\ncd \/mnt\/server\r\nwget -O ${SERVER_JARFILE} \"https:\/\/jitpack.io\/com\/github\/ViaVersion\/VIAaaS\/master-SNAPSHOT\/VIAaaS-master-SNAPSHOT-all.jar\"",
+            "container": "ghcr.io/pterodactyl/yolks:alpine",
```

Comment:
This is a base image. Not installer. This breaks the whole install process.

`ghcr.io/pterodactyl/installers:alpine` would be a correct image.

Reply:
jitpack link fixed